src/cascade/core/db.py

```diff
@@ -128,3 +130,14 @@ def latest_model_version(execution_context):
         else:
             raise RuntimeError(
                 f"No model version for modelable entity id {model_id} in database.")
+
+
+def dataframe_from_disk(path):
+    """ Load the file at `path` as a pandas dataframe.
+    """
+    if any(path.endswith(extension) for extension in [".hdf", ".h5", ".hdf5", ".he5"]):
```

Comment:
I've never seen he5. Where have you seen it?

Reply:
List of extensions used by hdf on wikipedia. I've not seen it either. We can take it out if you want to be tighter.